me.gladwell.eclipse.m2e.android/src/main/java/me/gladwell/eclipse/m2e/android/AndroidMavenLaunchConfigurationListener.java

```diff
@@ -63,6 +63,11 @@ public void launchConfigurationRemoved(ILaunchConfiguration configuration) {
     private void updateLaunchConfiguration(ILaunchConfiguration configuration) {
         try {
             String projectName = configuration.getAttribute(ATTR_PROJECT_NAME, (String) null);
+            
+            if (projectName == null) {
+                return;
```

Comment:
Good, simple solution but I'm a bit nervous about silently returning from this method.